.github/workflows/run-pr-tests.yaml

```
@@ -32,6 +32,10 @@ jobs:
     needs: build-noobaa-image
     uses: ./.github/workflows/ceph-s3-tests.yaml
 
+  ceph-ns-aws-s3-tests:
+    needs: build-noobaa-image
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)